R/html_vignette.R

```diff
@@ -48,20 +52,21 @@ html_vignette <- function(fig_width = 3,
                         quiet = TRUE)
     }
   }
+  dots <- list(...)
+  dots$theme <- NULL
+  dots$fig_width <- fig_width
+  dots$fig_height <- fig_height
+  dots$dev <- dev
+  dots$fig_retina <- NULL
+  dots$css <- css
+  dots$highlight <- "pygments"
 
   output_format(
     knitr = NULL,
     pandoc = NULL,
     df_print = df_print,
     pre_knit = pre_knit,
     keep_md = keep_md,
-    base_format = html_document(fig_width = fig_width,
-                                fig_height = fig_height,
-                                dev = dev,
-                                fig_retina = NULL,
-                                css = css,
-                                theme = NULL,
-                                highlight = "pygments",
-                                ...)
+    base_format = do.call(html_document, dots)
```

Comment:
Since you are not going to issue warnings, the rewrite here is no longer meaningful (I mean `do.call(FUN, list(...))`). Could you restore the changes in the function?

Reply:
The old version of the function cause an error when `theme` is supplied. The version in this pull request just ignores `theme` set by the user and compiles.

Do you prefer the error?

Reply:
Yes. I think it will be confusing if the `theme` option is silently ignored.